cli_reference/developer_cli_odo/creating-instances-of-services-managed-by-operators.adoc

```diff
@@ -7,10 +7,14 @@ toc::[]
 
 Operators are a method of packaging, deploying, and managing Kubernetes services. With `{odo-title}`, you can create instances of services from the Custom Resource Definitions (CRDs) provided by the Operators. You can then use these instances in your projects and link them to your components.
 
-To create services from an Operator, you must ensure that the Operator has valid values defined in its `metadata` to start the requested service. `{odo-title}` uses the `metadata.annotations.alm-examples` YAML file of an Operator to start
-the service. If this YAML has placeholder values or sample values, a service cannot start. You can modify the YAML file and start the service with the modified values. To learn how to modify YAML files and start services from it, go to xref:../../cli_reference/developer_cli_odo/creating-instances-of-services-managed-by-operators.adoc#creating-services-from-yaml-files_creating-instances-of-services-managed-by-operators[Creating services from YAML files].
+[NOTE]
+====
+You cannot install Operators with `{odo-title}` on your cluster. To install Operators, contact your cluster administrator or see xref:../../operators/user/olm-installing-operators-in-namespace.adoc#olm-installing-operators-in-namespace-prereqs[Installing Operators in your namespace]
+====
 
-== Prerequisites
+To create services from an Operator, you must ensure that the Operator has valid values defined in its `metadata` to start the requested service. `{odo-title}` uses the `metadata.annotations.alm-examples` YAML file of an Operator to start the service. If this YAML has placeholder values or sample values, a service cannot start. You can modify the YAML file and start the service with the modified values. To learn how to modify YAML files and start services from it, go to xref:../../cli_reference/developer_cli_odo/creating-instances-of-services-managed-by-operators.adoc#creating-services-from-yaml-files_creating-instances-of-services-managed-by-operators[Creating services from YAML files].
```

Comment:
You could consider swapping the order of the first three sentences. I suggest introducing the `metadata.annotations.alm-examples` YAML file, then introducing the requirements for the file. For example:

"`{odo-title}` uses the `metadata.annotations.alm-examples` YAML file of an Operator to start a service from an Operator. A service cannot start if this YAML file has placeholder or sample values. To start a service, you must ensure that the `metadata` section of the YAML has valid values."

Reply:
I suggest changing "To learn how to modify YAML files and start services from it" to "To learn how to modify YAML files and start services from them".